Elements/src/Geometry/Line.cs

```diff
@@ -1045,6 +1045,30 @@ public bool TryGetOverlap(Line line, out Line overlap)
             return true;
         }
 
+        /// <summary>
+        /// Creates new line with verticies of current and joined line
```

Comment:
"...with _vertices_ of current and joined..."

Reply:
Fixed. Sorry for this silly spelling, I added spell checker tool to my Visual Studio